core/src/main/scala/org/broadinstitute/dsde/workbench/hamm/db/JobTableQueries.scala

```diff
@@ -0,0 +1,111 @@
+package org.broadinstitute.dsde.workbench.hamm.db
+
+import java.sql.ResultSet
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.hamm.HammLogger
+import org.broadinstitute.dsde.workbench.hamm.model._
+import scalikejdbc._
+
```

Comment:
Scalike queries for the Job table:

- insert job
- get job
- get job cost
- get job's workflow's collection id (<--- why I added a foreign key on workflowId)